src/Components/Common/Paginator/Paginator.jsx

```diff
@@ -0,0 +1,73 @@
+import classes from './Paginator.module.scss';
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { http } from '../../../api/http';
+import { Button } from '../Button/Button';
+
+export const Paginator = ({
+                            page,
+                            pageSize,
+                            totalCount,
+                            setPage,
+                            setUsers,
+                            setTotalCount,
+                            portionSize = 10,
+                            toggleIsFetching
+                          }) => {
+  const onCurrentPage = (currentPage) => () => {
+    toggleIsFetching(true);
+    setPage(currentPage);
+    http.get(`users?page=${currentPage}&count=${pageSize}`)
```

Comment:
Do you have a handler in case of any service errors? =)

Reply:
The errors will be handled at a later stage of development.
Thank you for your comment!